packages/libs/sdk-mixins/src/mixins/themeMixin/themeMixin.ts

```diff
@@ -206,9 +206,15 @@ export const themeMixin = createSingletonMixin(
       async init() {
         await super.init?.();
 
-        this.#loadGlobalStyle();
-        this.#loadComponentsStyle();
         this.#onThemeChange();
+        try {
+          await Promise.all([
+            this.#loadGlobalStyle(),
+            this.#loadComponentsStyle(),
+          ]);
+        } catch (e) {
+          this.logger.error('Failed to load theme', e);
```

Comment:
do we want to throw the error as well?

Reply:
No need to catch here
committed by mistake